src/frontend/lib/views/addfield/add_field_screen.dart

```diff
@@ -0,0 +1,97 @@
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:flutter/material.dart';
+import 'package:frontend/providers/new_field_provider.dart';
+import 'package:frontend/stores/fields_store.dart';
+import 'package:frontend/views/addfield/add_field_details_screen.dart';
+import 'package:frontend/views/addfield/widgets/add_field_map.dart';
+import 'package:frontend/views/addfield/widgets/bottom_bar_add_field.dart';
+import 'package:frontend/widgets/terrafarm_app_bar.dart';
+import 'package:frontend/views/map.dart';
+import 'package:provider/provider.dart';
+
+import '../../models/field_model.dart';
+import '../../models/user_model.dart';
+
+class AddFieldScreen extends StatefulWidget {
+  const AddFieldScreen({super.key});
+
+  @override
+  State<AddFieldScreen> createState() => _AddFieldScreenState();
+}
+
+class _AddFieldScreenState extends State<AddFieldScreen> {
+
+  @override
+  void initState() {
+    super.initState();
+    Provider.of<NewFieldProvider>(context, listen: false).clearGeoPoints();
+  }
+  @override
+  Widget build(BuildContext context) {
+    // Curent user that is logged in
+    final user = Provider.of<UserModel>(context);
+
+    // Stream listens for updates to 'Fields' collection
+    return StreamProvider<List<FieldModel>>.value(
+      value: FieldsStore(userId: user.uid).fields,
+      initialData: const [],
+      child: Scaffold(
+        backgroundColor: Colors.white,
+        appBar: TerraFarmAppBar(
+          onPressed: () {
+            Navigator.pop(context);
+          },
+          text: 'Add new field',
+        ),
+        body: Padding(
+          padding: const EdgeInsets.all(8.0),
+          child: Center(
+            child: Flex(
+              direction: Axis.vertical,
+              children: [
+                Text(
+                  "Draw borders for your desired field to be added.",
```

Comment:
It could be a bit ambigiuous for users; remember, they're not technically strong. How about:
`Define the field boundaries by selecting points around the field`

Reply:
Ok sure! I just copied the text from the design